client/components/filters/date/style.scss

```diff
@@ -95,6 +101,7 @@
 
 .woocommerce-filters-date__button {
 	justify-content: center;
-	width: 50%;
-	margin: 0 1em;
+	width: 40%;
+	height: 34px !important;
```

Comment:
These rules work without needing to use the !important.

```
.woocommerce-filters-date__button {
	justify-content: center;
	width: 40%;
	&.components-button {
		margin: 0 $gap-small;
		&.is-button {
			height: 34px;
			line-height: 32px;
		}
	}
}
```

Reply:
It is a matter of specificity, so that would work, but it's a little unclear that those styles always apply - it would be better to keep everything in one rule-block. We could nest it in `.woocommerce-filters-date__button-group`, and add the extra specificity of `is-button`:

```
.woocommerce-filters-date__button-group {
	…
	.woocommerce-filters-date__button.is-button {
		justify-content: center;
		width: 40%;
		height: 34px !important;
		margin: 0 0.75em !important;
	}
}
```